config.py

```diff
@@ -0,0 +1,20 @@
+import os
+from dotenv import load_dotenv
+
+APP_DIR = os.path.abspath(os.path.dirname(__file__))
```

Comment:
I still think that this is overly complicated. We'll have to handle most stuff from central scripts anyways if we want to take advantage of automations so this is not necessary.

Reply:
i am still a bit confused about this tbh. what is the advantage of using the directory *from where the command is executed* vs *where the file is located*? this on-liner is something i see in many projects and i accepted it as "idiomatic" and just defaulted to using it. i don't really have an opinion about this, but it seems more deterministic and less prone to unexpected errors. if it complicates things for us, we should change it, but you probably know something that i don't :)

Reply:
I've never worked on a project where starting / stopping has not been implemented via scripts / commands that are run from `app_dir` by default is all. In the context of a web app, this line adds a safety net for a situation that should neither happen nor be encouraged to happen as it indicates that scripts / commands are scattered around the codebase when they shouldn't be.